packages/blockchain-info-components/src/Fonts/Icomoon/fonts/icomoon.svg

```diff
@@ -82,6 +82,8 @@
 <glyph unicode="&#xe949;" glyph-name="lock" d="M739.962 556.8h-41.448v60.58c0 103.48-82.895 188.76-185.996 189.020s-187.032-83.85-187.032-187.2v-62.4h-41.448c-34.324 0-62.171-27.95-62.171-62.4v-291.2c0-34.45 27.848-62.4 62.171-62.4h455.924c34.324 0 62.171 27.95 62.171 62.4v291.2c0 34.45-27.848 62.4-62.171 62.4zM387.657 619.2c0 68.77 55.825 124.8 124.343 124.8s124.343-56.030 124.343-124.8v-62.4h-248.686v62.4zM739.962 203.2h-455.924v291.2h455.924v-291.2z" />
 <glyph unicode="&#xe94a;" glyph-name="usdt" d="M512-64c282.771 0 512 229.229 512 512s-229.229 512-512 512c-282.77 0-512-229.23-512-512s229.23-512 512-512zM573.504 403.744v0.064c-3.52-0.256-21.664-1.344-62.144-1.344-32.32 0-55.072 0.96-63.072 1.344v-0.096c-124.416 5.472-217.28 27.136-217.28 53.056 0 25.888 92.864 47.552 217.28 53.12v-84.608c8.128-0.576 31.424-1.952 63.616-1.952 38.624 0 57.984 1.6 61.6 1.92v84.576c124.16-5.536 216.8-27.2 216.8-53.056 0-25.92-92.64-47.52-216.8-53.024zM573.504 518.624v75.712h173.248v115.456h-471.712v-115.456h173.248v-75.68c-140.8-6.464-246.688-34.368-246.688-67.776s105.888-61.28 246.688-67.776v-242.624h125.216v242.688c140.576 6.464 246.208 34.336 246.208 67.712s-105.632 61.248-246.208 67.744z" />
+<glyph unicode="&#xe94b;" glyph-name="gbp" d="M0 667.429c0 161.583 130.989 292.571 292.571 292.571h438.857c161.584 0 292.571-130.989 292.571-292.571v-438.857c0-161.584-130.988-292.571-292.571-292.571h-438.857c-161.583 0-292.571 130.988-292.571 292.571v438.857zM354.784 485.921v74.624c0 117.631 71.010 191.455 194.402 191.455 93.824 0 146.238-39.744 186.814-90.88l-79.455-60.8c-30.435 37.28-59.202 59.2-106.496 59.2-56.642 0-90.463-38.944-90.463-101.44v-72.159h210.494v-84.385h-210.494v-138.719h274.717v-86.817h-446.303v63.265l66.784 19.489v142.782h-66.784v84.385h66.784z" />
+<glyph unicode="&#xe94d;" glyph-name="eur" d="M0 667.429c0 161.583 130.989 292.571 292.571 292.571h438.857c161.584 0 292.571-130.989 292.571-292.571v-438.857c0-161.584-130.988-292.571-292.571-292.571h-438.857c-161.583 0-292.571 130.988-292.571 292.571v438.857zM645.84 533.687h-209.156c19.712 74.739 67.346 110.876 128.669 110.876 47.909 0 84.868-22.449 111.148-43.255l38.327 81.582c-40.517 33.947-90.891 53.11-149.475 53.11-116.352 0-204.229-74.464-231.058-202.313h-56.669l-21.627-58.309h70.631c-0.547-9.037-0.821-18.344-0.821-27.926 0-9.033 0.274-18.066 0.821-26.829h-49.004l-21.627-59.132h78.297c26.829-127.572 114.706-201.49 231.058-201.49 56.393 0 108.683 17.247 147.833 51.193l-38.875 81.584c-23.815-19.163-60.774-41.34-108.957-41.34-61.597 0-108.957 35.317-128.669 110.054h130.311l28.197 59.132h-167.541c-0.549 8.488-0.823 17.521-0.823 26.829 0 9.582 0.274 18.889 0.823 27.926h191.634l26.555 58.309z" />
```

Comment:
I'm just curious where usd will came?

Reply:
I extract these as svgs from figma then use icomoon.app to generate a font